app/backend/src/couchers/servicers/jail.py

```diff
@@ -14,15 +14,13 @@
 
 
 def _get_jail_info(session, user):
-    pending_notes = (
-        session.execute(select(ModNote).where(ModNote.user_id == user.id).where(ModNote.is_pending)).scalars().all()
-    )
     res = jail_pb2.JailInfoRes(
-        has_not_accepted_tos=user.accepted_tos < TOS_VERSION,
+        has_not_accepted_tos=user.jailed_missing_tos,
         has_not_added_location=user.is_missing_location,
-        has_not_accepted_community_guidelines=user.accepted_community_guidelines < GUIDELINES_VERSION,
-        has_pending_mod_notes=len(pending_notes) > 0,
-        pending_mod_notes=[mod_note_to_pb(note) for note in pending_notes],
+        has_not_accepted_community_guidelines=user.jailed_missing_community_guidelines,
+        has_pending_mod_notes=user.jailed_pending_mod_notes,
+        pending_mod_notes=[mod_note_to_pb(note) for note in user.mod_notes.where(ModNote.is_pending)],
```

Comment:
These are unrelated, or? We just sneaking them in now or are they also a type of activeness probe?

Reply:
Unrelated. No behavior here was changed, I just cleaned up the code!